### DISCUSSION TITLE
Any example about bulk emails

### DISCUSSION THREAD

**Discussion by onlinejudge95** [sabuhish/fastapi-mail#74] [General]
The repo does not contain any example for bulk email as described in the README.md.
Can you please point out to some.

**Comment by onlinejudge95** [comment]
Passing multiple emails to receiver attributes makes all of them visible in the `To` section. I was expecting if we can send emails to each user individually but instead of sending all emails one by one send them in bulk to the SMTP server which buffers them individually.

**Reply by Turall** [reply]
yes you are right, if you have any PR on this, we will be happy to consider and discuss it
